templates/email/subscription-errored.live

```diff
@@ -0,0 +1,11 @@
+{% extends "email/email" %}
```

Comment:
Why is this a `.live` file when all the other files in this email dir are `.hypr`?